vagrant/ansible/roles/samba-glusterfs.setup/templates/smb.conf.j2

```diff
@@ -1,19 +1,27 @@
 [global]
+workgroup = MYGROUP
+server string = Samba Server Version %v
+log file = /var/log/samba/log.%m
+max log size = 0
+security = user
+passdb backend = tdbsam
 netbios name = {{ samba_netbios_name }}
 clustering = yes
-passdb backend = tdbsam
 
-host msdfs = yes
-msdfs root = yes
-msdfs shuffle referrals = yes
+max protocol = SMB3
+kernel share modes = no
+kernel change notify = no
 
-client signing = auto
+kernel oplocks = no
+posix locking = no
 
-kernel share modes = No
-map archive = No
-posix locking = No
+map archive = no
+map hidden = no
+map read only = no
+map system = no
+store dos attributes = yes
 
 load printers = no
-printcap name = /dev/null
```

Comment:
I think `printcap name = /dev/null` should be kept to completely disable printing:

https://serverfault.com/questions/207510/how-do-you-disable-smb-printing-support
https://lists.samba.org/archive/samba/2006-February/117184.html

Reply:
Done